zerver/lib/onboarding.py

```diff
@@ -111,18 +113,119 @@ def send_welcome_bot_response(send_request: SendMessageRequest) -> None:
+            "and edit your [profile information](https://zulip.com/help/edit-your-profile)."
+        )
+    elif any(x in human_response for x in ["mode", "theme"]):
+        return _(
+            "Go to [Display settings](#settings/display-settings) "
+            "to [switch between light and dark mode](https://zulip.com/help/night-mode), "
+            "[pick your favorite emoji theme](https://zulip.com/help/emoji-and-emoticons#change-your-emoji-set), "
+            "[change your language](https://zulip.com/help/change-your-language), "
+            "and make other tweaks to your Zulip experience."
+        )
+    elif any(x in human_response for x in ["streams", "channels"]):
+        return "".join(
+            [
+                _(
+                    "In Zulip, streams [determine who gets a message](https://zulip.com/help/streams-and-topics). "
+                    "They are similar to channels in other apps."
+                )
+                + "\n\n",
+                _("[Browse and subscribe to streams](#streams/all)."),
+            ]
+        )
+    elif "topics" in human_response:
+        return "".join(
+            [
+                _(
+                    "In Zulip, topics [tell you what a message is about](https://zulip.com/help/streams-and-topics). "
+                    "They are light-weight subjects, very similar to the subject line of an email."
+                )
+                + "\n\n",
+                _(
+                    "Check out [Recent topics](#recent_topics) to see what's happening! "
+                    'You can return to this conversation by clicking "Private messages" in the upper left.'
+                ),
+            ]
         )
-        internal_send_private_message(welcome_bot, send_request.message.sender, content)
+    elif "shortcuts" in human_response:
+        return "".join(
+            [
+                _(
+                    "Zulip's [keyboard shortcuts](#keyboard-shortcuts) "
+                    "let you navigate the app quickly and efficiently."
+                )
+                + "\n\n",
+                _("Press `?` any time to see a [cheat sheet](#keyboard-shortcuts)."),
+            ]
+        )
+    elif "format" in human_response:
+        return "".join(
+            [
+                _(
+                    "Zulip uses [Markdown](https://zulip.com/help/format-your-message-using-markdown), "
+                    "an intuitive format for **bold**, *italics*, bulleted lists, and more. "
+                    "Click [here](#message-formatting) for a cheat sheet."
+                )
+                + "\n\n",
+                _(
+                    "Check out our [messaging tips](https://zulip.com/help/messaging-tips) "
+                    "to learn about emoji reactions, code blocks and much more!"
+                ),
+            ]
+        )
+    elif any(x in human_response for x in ["help", "?"]):
```

Comment:
Checking just "in" is too wide -- otherwise any message containing `?` will get this response, regardless of where it appears.  I think we want all these tests to be more like `human_response.lower() in []`.